patterns/bottlerocket/README.md

```diff
@@ -0,0 +1,151 @@
+# Bottlerocket with Bottlerocket Update Operator
+
+This pattern demostrates how to deploy Amazon EKS Clusters using [Bottlerocket OS](https://aws.amazon.com/bottlerocket/) for Managed Node Groups (MNG) and [Karpenter](https://karpenter.sh/), using [Bottlerocket Update Operator (BRUPOP)](https://github.com/bottlerocket-os/bottlerocket-update-operator) to manage CVE patches automatically at the Node OS level. The BROPUP doesn't work with minor or major upgrades, just patch level.
+
+## Deploy
+
+See [here](https://aws-ia.github.io/terraform-aws-eks-blueprints/getting-started/#prerequisites) for the prerequisites and steps to deploy this pattern.
+
+## Validate
+
+* List all Nodes in the cluster. You should see three Nodes that belongs to the defined MNG, and should be in the `v1.27.4-eks-cedffd4` version since we are using a specific AMI ID to teste the BRUPOP.
```

Comment:
```suggestion
* List all Nodes in the cluster. You should see three Nodes that belongs to the defined MNG, and should be in the `v1.27.4-eks-cedffd4` version since we are using a specific AMI ID to test the BRUPOP.
```